examples/build_file_generation/__main__.py

```diff
@@ -1,4 +1,19 @@
+# Copyright 2022 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 from __init__ import main
 
+"""Print out the bazel-contrib website"""
 if __name__ == "__main__":
-    main("https://example.com")
+    main("https://bazel-contrib.github.io/SIG-rules-authors/")
```

Comment:
Maybe on a separate PR, we should eliminate tests that reach out to the internet.

Reply:
Actually I would rather just knock out a code refactor here.  I need to complete the refactor as it will impact my work on the tutorial.

I am thinking I will make a small flask microservice. For the tutorial I need an external dependency and some code that I can write a unit test for.  I will use flask as the external dep and write a random number generator that I will test.  The flask app will return a random number.

Thoughts?  BTW thanks for the awesome code review :)

Reply:
Sounds good. Anything that you think would be good for the tutorial.